### ISSUE TITLE
Issue with some modifiers

### ISSUE THREAD

**Issue by agahowa23** [LoneGazebo/Community-Patch-DLL#6633]
_1. Mod version (i.e Date - 4/17): 7-29-1 (I downloaded the automatic installer from the forum 2 weeks ago)

_2. Mod list (if using Vox Populi only, leave blank):_

_3. Error description: At some point of the game (I didn't want to look for the exact turn cause I know the issue will surely happen again even if I load a previous save, and I think I noticed it too late so the saves are gone) one of my cities went crazy. Something happen (an event, a building finished, who knows...) that made my city useless, it generates negative production, negative food, negative gold... it is a mess and it basically ruined the game. I noticed that some buildings had "strange numbers", over the thousands I mean. For example, the Granary were generating hundreds of thousands of food, while others were generating hundreds of thousands of production... I sold those buildings, but there's one I cannot sell and is the only one, now, that's generating a ludicrous percentage of something I don't understand. It is the Customs House, which, for some reason I can't sell. It is a modifiers issue I think. I wanna know, is there a way I can manually change those values or to correct them? I even thought of razing the city, but I can't because I founded it. Please help me...
I can't upload the log files cause I had logging disabled and as a matter of fact, the game isn't crashing, it runs fine, just with a broken city... Also, I can't upload the saved game cause I don't  know when the error occurred and the save is probably lost.
I mean... I will upload those files, but I think they will be useless.

4. Screenshots (optional):

![Look at the Customs House](https://user-images.githubusercontent.com/39000549/83362476-d8e6c380-a34e-11ea-9dcf-410f39127fe7.jpg)
![Look at Production](https://user-images.githubusercontent.com/39000549/83362477-d97f5a00-a34e-11ea-8379-c3d32e241c26.jpg)
![Look at Food](https://user-images.githubusercontent.com/39000549/83362479-dab08700-a34e-11ea-9cbb-64e815f4b57f.jpg)

[CivV.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/4708403/CivV.zip)

**Comment by RecursiveStar** [comment]
@agahowa23 Are you using any other mods?

**Comment by agahowa23** [comment]
First I want to thank you for your answer RecursiveStar! I know it must be freaking boring to help noobs with this. So thank you for your time.
No, I am not using any other mod, in fact, the Vox Populi is the first I try. I own the legit steam version also. I will follow your suggestion. To install the beta version should I do a clean setup of the game first?

**Comment by RecursiveStar** [comment]
@agahowa23 
Download the autoinstaller (.exe file) from here: https://mega.nz/folder/2Jdw3aQT#Z49gV7KAyMq6YiG5BD9jaA

Since you're experiencing issues, try clearing your cache and ModUserData folders (located at My Games\Sid Meier's Civilization 5\cache and ModUserData, respectively.

If you're still experiencing issues with the latest version I'd recommend asking @Vyyt02 for help to make sure it isn't a problem with your installation.

**Comment by azum4roll** [comment]
Make sure you're UNsubscribed to the Steam Workshop version of the Community Patch.

**Comment by agahowa23** [comment]
Thank you all so much! Will do.

**Comment by Vyyt02** [comment]
Did someone summon me? :-)

1) I vaguely remember that a very old VP version was sometimes doing funny things with numbers like this, but I am pretty sure it was way older than the stable release 7-29-1. Yeah, I agree to installing the latest beta (4-17) should fix this. If it doesn't then we'll all at least be playing the same game and it will be easier to debug this :-) Just to make sure - when you run the autoinstaller.exe, do NOT delete any files or perform any manual action.

2) The game does not have to crash to create logs. I think you are mixing it with the crash minidump

3) I can also see some weirdness in the tooltips - the TXT_KEY_YIELD_etc. should not be there. It comes from events. When you start a new game via advanced setup and turn all the events off, will these disappear?

**Comment by agahowa23** [comment]
Hi man!
1. Yeah, it made some shenanigans, but now I installed the beta and everything seems to be working fine. Although I already deleted the mods and the moduserdata folder manually before installing the new version :/
2. Oh, I know, but I hadn't enabled logging at that moment so I had no useful logs... I think.
3. I can't try that now because I already installed the beta VP, but I will if it happens again. As a matter of fact I thought the same, that events were doing that.
4. I thank you so much!!! Thanks for your time and good will!
